### ISSUE TITLE
Question: Any reasons Not handling slow call?

### ISSUE THREAD

**Issue by proost** [failsafe-go/failsafe-go#53]
Many circuit breaker (ex. resilience4j, sentinel) handle slow call.  But failsafe-go deosn't. Are there reasons not hanlding slow call?

**Comment by jhalterman** [comment]
No reason - I'm open to adding this. Can you share how you would like this to work? What settings would you like to have, and how would you use this?

**Comment by jhalterman** [comment]
Yes, you can compose different breakers with different errors and thresholds, and in this case, you can compose them around a timeout which handles slow executions.
